Utils/azuretools-core/src/com/microsoft/azuretools/authmanage/AuthMethodManager.java

```diff
@@ -247,4 +254,10 @@ private static AuthMethodDetails loadSettings() {
             return new AuthMethodDetails();
         }
     }
+
+    private static AzureToolkitRuntimeException newAzureToolkitRuntimeExceptionWhenNotSignedIn(final String sid) {
```

Comment:
we may create a ToolkitNotSignInException which extends AzureToolkitRuntimeException

Reply:
It is a general consideration, better to leave it to other dedicated PR